adobe_vipm/flows/sync.py

```diff
@@ -857,25 +922,35 @@ def sync_deployments_prices(
         main_agreement: Main MPT agreement.
         customer: Adobe customer.
         customer_deployments: Adobe customer deployments.
-        dry_run: If True doesn't update agreement.
-        sync_prices: If True also sync prices. Keep in mind dry_run parameter also.
+        customer_subscriptions: list of subscriptions for customer from Adobe.
+        dry_run: Run command in a dry run mode.
+        sync_prices: If True also sync prices.
     """
     if not customer_deployments:
         return
 
     deployment_agreements = get_agreements_by_customer_deployments(
         mpt_client,
-        Param.DEPLOYMENT_ID,
+        Param.DEPLOYMENT_ID.value,
         [deployment["deploymentId"] for deployment in customer_deployments],
     )
 
+    _process_orphaned_deployment_subscriptions(
+        adobe_client,
+        main_agreement["authorization"]["id"],
```

Comment:
@alephsur can you please confirm it is OK to pass here the`main_agreement["authorization"]["id"]`? Is the authorisation id the same for all subscriptions of a specific customer in a context of his all deployments and main agreement? # @d3rky